### PR TITLE
Bug: Schedules on pr deployments...

### PR DESCRIPTION
**Pull request by haakonvt**

### CODE REVIEW

File: src/config.py [hunk 1]
```diff
@@ -154,15 +155,6 @@ def valid_secret(cls, value):
...
         return value
 
-    @validator("remove_only")
-    def valid_remove_only(cls, value):
-        if value is None:
-            return False
-        if isinstance(value, bool):
-            return value
-        elif isinstance(value, str):
-            return value.lower().strip() == "true"
```
Comment (haakonvt) [inline src/config.py:164]: Pydantic already did this (almost)

**Review by pavelzubarev — COMMENTED** [review]
Should we check in that case that `attach_schedule` makes `schedule_file` a required field?

**Comment by haakonvt** [comment]
> Should we check in that case that `attach_schedule` makes `schedule_file` a required field?

How to distinguish between "passed as True" vs "not passed, default (True) passed"?

**Comment by pavelzubarev** [comment]
> > Should we check in that case that `attach_schedule` makes `schedule_file` a required field?
> 
> How to distinguish between "passed as True" vs "not passed, default (True) passed"?

Shall we do that at all? I mean if it's default and set to True, you should expect the schedule be in place. I think regardless it comes by default or intentionally (aka I like explicit calls) we should check that file is specified.

The alternative could be that if the file is not specified => we don't attach the schedule.

**Review by pavelzubarev — APPROVED** [review]

**Pull request merged**